build.win32x86/common/Makefile.msvc.tools

```diff
@@ -53,16 +53,17 @@ ifdef MSVC
 MEMACCESS:=-DUSE_INLINE_MEMORY_ACCESSORS=1
 endif
 
-# Set minimum version to WindowsXP (see /cygwin/usr/include/w32api/w32api.h)
-WINVER:=-D_WIN32_WINNT=0x0501 -DWINVER=0x0501
+# Set minimum version to Windows 8 (see /cygwin/usr/include/w32api/w32api.h)
+WINVER:=-D_WIN32_WINNT=0x0602 -DWINVER=0x0602
```

Comment:
As suggested on the mailing list, I would just use 0x0600

See also https://docs.microsoft.com/en-us/cpp/porting/modifying-winver-and-win32-winnt?view=vs-2019 for the list of _WIN32_WINNT version.

Reply:
How would you feel about setting minimum supported version to 8.0?  I want to do this because of the GetSystemTimePreciseAsFileTime function, which appears in 8.0 nd neatly solves the time issues in the win32 VM (use of the millisecond clock to derive greater precision than the 15ms tick in GetSystemTimeAsFileTime causes significant direct over the course of several days).

Reply:
We can have `0x6000` (Vista) _and_ use that function. Ping me in a week or so, when I'm back from vacation